src/components/Item.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { View, Image, Text, ImageProps, StyleSheet } from "react-native";
+import Octicons from "@expo/vector-icons/Octicons";
+import { Typography } from "@/theme/typography";
+import { ColorTheme } from "@/theme/colors";
+
+export const Item = (props: ItemProps) => {
+  const { heading, subtitle, body, headingSize, ...avatarProps } = props;
+  const bodyProps = { heading, subtitle, body, headingSize };
+  return (
+    <View style={ItemStyles.row}>
+      <ItemAvatar {...avatarProps} />
+      <ItemBody {...bodyProps} />
+    </View>
+  );
+};
+
+export const ItemStack = (props: ItemStackProps) => {
+  const { images, ...bodyProps } = props;
+  return (
+    <View>
+      <View style={ItemStyles.row}>
+        {images.map((imageProps, index) => (
+          <ItemAvatar key={index} {...imageProps} />
+        ))}
+      </View>
+      <ItemBody {...bodyProps} />
+    </View>
+  );
+};
+
+export const ItemAvatar = (props) => {
+  const { source, iconName, badgeName } = props;
+  if (!source && !iconName) {
+    throw new Error("Must supply either `source` or `iconName`");
+  }
+  return (
+    <View style={ItemStyles.iconAvatarContainer}>
+      {source ? (
+        <Image source={source} style={ItemStyles.iconAvatarImage} />
+      ) : (
+        <Octicons name={iconName} size={20} style={ItemStyles.iconAvatar} />
+      )}
+      {badgeName ? (
+        <View style={ItemStyles.badgeAvatarContainer}>
+          <Octicons name={badgeName} size={12} style={ItemStyles.badgeAvatar} />
+        </View>
+      ) : null}
+    </View>
+  );
+};
+
+export const ItemBody = (props) => {
+  const { heading, subtitle, body, headingSize } = props;
+  return (
+    <View>
+      <Text style={Typography[headingSize ?? "body1"]}>{heading}</Text>
```

Comment:
not obvious to me that `headingSize` means a style of typography. `headingStyle` ?